packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -356,6 +359,12 @@ class DatePickerDialog extends StatefulWidget {
   ///    Flutter.
   final String? restorationId;
 
+  
+  /// Called when the [DatePickerDialog] is toggled between 
+  /// [DatePickerEntryMode.calendar],[DatePickerEntryMode.input] and return 
```

Comment:
```suggestion
  /// [DatePickerEntryMode.calendar], [DatePickerEntryMode.input].
```

This is a void callback, so it doesn't return a value.

It would still help to add a sentence or two to explain the likely use case for this callback.

Reply:
There's still no explanatory sentence here. There's a use case in the PR's description, so I'd suggest going with that:
```dart
/// 
/// An example of how this callback might be used is an app that saves the 
/// user's preferred entry mode and uses it to initialize the `initialEntryMode` parameter
/// the next time the date picker is shown.
```

Reply:
Sure I have added the same comment below